apps/andi/lib/andi_web/live/edit_live_view/url_form.ex

```diff
@@ -229,6 +232,16 @@ defmodule AndiWeb.EditLiveView.UrlForm do
 
   defp status_class(%{status: status}) when status in 200..399, do: "test-status__code--good"
   defp status_class(%{status: _}), do: "test-status__code--bad"
+  defp status_tooltip(%{status: status}) when status in 200..399, do: status_tooltip(%{status: status}, "hidden")
+
+  defp status_tooltip(%{status: status}, modifier \\ "shown") do
+    assigns = %{
+      description: HttpStatusDescriptions.get(status),
+      modifier: modifier
+    }
+
+    ~E(<sup class="test-status__tooltip-wrapper"><i phx-hook="addTooltip" data-tooltip-content="<%= @description %>" class="material-icons-outlined test-status__tooltip--<%= @modifier %>">info</i></sup>)
+  end
```

Comment:
this is a cool pattern, we could probably stand to do this where we have some checks (like for visibility) at the beginning of our render functions

Reply:
Yeah, true. The only gotcha there would be hidden fields for forms (possibly). I can't recall if non-hidden form fields inside a non-displayed parent get send to the backend or not.